src/lib.rs

```diff
@@ -663,7 +763,10 @@ decl_module! {
                 created_at : Self::current_block_and_time(),
                 deleted: false,
                 archived: false,
-                parent_id: parent,
+                num_direct_subcategories: 0,
+                num_direct_unmoderated_threads: 0,
```

Comment:
Seems like a typo here:
```suggestion
                num_direct_unmoderated_threads: 1,
                num_direct_moderated_threads: 0,
```

Reply:
Fixed.